### PR TITLE
Style time

### PR DESCRIPTION
**Pull request by perryr16**
greens and two pane screen

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -17,19 +17,63 @@
...
+   height: 80%;
+   top:0;
+   left: 0;
+
+   margin-top: 80px;
+   padding: 10px;
+   border-left: solid 8px #0077A6;
+ }
+ .content-right{
+   background-color:#185946;
```
Comment (perryr16) [inline app/assets/stylesheets/application.css:53]: content right is a section that displays content on right of screen

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -17,19 +17,63 @@
...
    margin-top: 80px;
+   margin-left: 10vw;
+   margin-right: 10vw;
+
+   padding: 10px;
  }
- .content {
-   margin-left: 20px;
+ .content-left{
+   background-color:#134234;
```
Comment (perryr16) [inline app/assets/stylesheets/application.css:40]: content left is a section that displays content on left of screen

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -108,11 +152,21 @@
   left: 50%;
   height: 20vh;
 }
+.pet-image-small-float {
```
Comment (perryr16) [inline app/assets/stylesheets/application.css:155]: did not work... this can be deleted

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -9,25 +9,27 @@
   </head>
 
   <body>
-
-    <%= yield %>
     <div class="background">
-      <img src="https://wallpaperaccess.com/full/783057.jpg" alt="kitten">
+      <img src="https://tg-cdn.azureedge.net/sites/default/files/2019-10/meadow-3375052_1920-%281%29.jpg" alt="grass">
```
Comment (perryr16) [inline app/views/layouts/application.html.erb:13]: grass. not pug

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (muydanny) [reply app/assets/stylesheets/application.css:53]: We dont mess around out here

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (muydanny) [reply app/views/layouts/application.html.erb:13]: RIP

**Pull request merged**